### DISCUSSION TITLE
How to run whisper faster xxl pls

### DISCUSSION THREAD

**Discussion by Policing2024** [SubtitleEdit/subtitleedit#8203] [Ideas]
How to run whisper faster xxl pls

**Comment by Purfview** [comment]
Maybe @niksedk will add support for it.

Currently you can rename exe file to a regular `whisper-faster.exe` to make it delectable by SE.

https://github.com/Purfview/whisper-standalone-win/releases/tag/Faster-Whisper-XXL

**Reply by Policing2024** [reply]
I renamed to whisper-faster.exe
But it shows no text found error please

**Reply by Purfview** [reply]
Check what is in "whisper_log.txt" or better try to run it directly.

**Reply by Policing2024** [reply]
How to run it directly please?

**Reply by Purfview** [reply]
Same as any other command line tool, run it in any console/terminal of your liking.

https://www.google.com/search?q=how+to+run+command+line+tool+in+windows
